guide/webots-faq.md

```diff
@@ -2,7 +2,7 @@
 
 This chapter is a selection of frequently asked questions found on the [Webots forum](http://www.cyberbotics.com/forum).
 You may find additional information directly in the group.
-Other useful sources of information about Webots include: [Webots Reference Manual](http://www.cyberbotics.com/reference/) and [Cyberbotics' Robot Curriculum](http://en.wikibooks.org/wiki/Cyberbotics'_Robot_Curriculum).
+Other useful sources of information about Webots include: [Webots Reference Manual](../reference/index.md) and [Cyberbotics' Robot Curriculum](http://en.wikibooks.org/wiki/Cyberbotics'_Robot_Curriculum).
```

Comment:
I would say "... include the Webots Reference Manual and the Cyberbotics' Robot Curriculum".